### PR TITLE
Two body time

### PR DESCRIPTION
**Pull request by jake163**

### CODE REVIEW

File: geometries/Hexamer_Prism.xyz [hunk 1]
```diff
@@ -3,12 +3,12 @@ E(RHF/6-31G(d,p))=-456.2206685
  O                   1.526924   -0.407825    1.467087
  H                   0.615293   -0.694811    1.569578
  H                   2.001971   -0.712098    2.222674
- O                   0.831994    2.016713   -0.073306
- H                   1.169971    1.534473    0.670375
- H                   1.181010    1.553886   -0.823001
```
Comment (jake163) [inline geometries/Hexamer_Prism.xyz:8]: I rearranged some waters so that A is near B, B is near C, etc.

**Comment by jake163** [comment]
I debugged `xyz-to-coords.py` and pushed here (I called the commit 'debug'). I'm not sure what the issue is.

File: geometries/Hexamer_Prism.xyz (hunk 1 above)
Comment (jake163) [reply geometries/Hexamer_Prism.xyz:8]: The order of the monomers is important in the combinatorial making of the dimers, and the making of rho.

Comment (ryanmrichard) [reply geometries/Hexamer_Prism.xyz:8]: I'm a little concerned by this statement because the order shouldn't matter. In what way does the order matter?

Comment (jake163) [reply geometries/Hexamer_Prism.xyz:8]: Our guess rho needs to be ordered (in terms of the dimer subblocks), yes? I mentioned that rho is constructed using the index of the dimer densities (it is a list of lists with repeat densities). That relies on an ordered dimer input (again, a list of lists but no repeat densities). This, in turn, is made with an ordered monomers list by `itertools.combinations`. 

All I mean by the first comment is that `mol[0:3]`, which corresponds to monomer A, is indeed next to `mol[3:6]`, which corresponds to monomer B. With regard to the `.xyz` file, I didn't want monomer B to be separated from A by say two other monomers, because then it wouldn't be B, it would be D (i.e. `mol[9:12]`).

In other words, my understanding is that diagonal of the rho guess is essentially the order of the monomers in the structure that the scf energy is being calculated for. If this is not true then I am indeed misunderstood.

Would you mind doing a zoom call?

Comment (jake163) [reply geometries/Hexamer_Prism.xyz:8]: At this point I think it would benefit both of us, but it's your call.

Comment (jake163) [reply geometries/Hexamer_Prism.xyz:8]: In other words, my understanding is that diagonal of the rho guess is essentially the order of the monomers in the structure that the scf energy is being calculated for. If this is not true then I am indeed misunderstood.
